src/components/Causes/css.js

```diff
@@ -0,0 +1,84 @@
+import styled from 'styled-components';
+import EventPhoto from '../../images/womenintech-6.jpg';
+
+const CausesSection = styled.section`
+	@media (min-width: 850px) {
+		background-color: #ffffff;
+	}
+
+	@media (max-width: 849px) {
+		background-image: url(${EventPhoto});
+		background-position: center center;
+		background-size: cover;
+	}
+`;
+
+const CausesPhoto = styled.section`
+	@media (min-width: 850px) {
+		background-image: url(${EventPhoto});
+		background-size: cover;
+		position: relative;
+		height: 500px;
+		width: 800px;
+		left: 5rem;
+		top: 6rem;
+	}
+`;
+
+const CausesTextSection = styled.section`
+	@media (min-width: 1025px) {
+		background-color: #ffffff;
+		position: relative;
+		height: 450px;
+		width: 700px;
+		margin-left: 32rem;
+		top: -11rem;
+		margin-top: -12.5rem;
+		margin-bottom: -2.5rem;
+		padding: 3rem;
+	}
+
+	@media (min-width: 850px) and (max-width: 1024px) {
+		background-color: #ffffff;
+		position: relative;
+		height: 450px;
+		width: 700px;
+		margin-left: auto;
+		top: -11rem;
+		margin-top: -12.5rem;
+		margin-bottom: -2.5rem;
+		padding: 3rem;
+	}
+
+	@media (max-width: 849px) {
+		background: rgba(255, 255, 255, 0.75);
+		padding: 3rem;
+		text-align: center;
+	}
+`;
+
+const CausesH1 = styled.header`
```

Comment:
I would change this to an H2 :) The reason why I wouldn't say H1 is generally we reserve H1 for the Hero/header heading at the very top. It's normal to use H2 for section headers with H3 as subheadings in each section. 
< header > itself should be reserved for the top hero header/navigation area while everything below it in the sections would to in the <main></main> section until you hit the footer.